config/initializers/filter_parameter_logging.rb

```diff
@@ -51,6 +51,7 @@
   service
   use_v2
   line
+  payload
```

Comment:
This will let in the `payload` through without filtering across the board, which I don't think we want.

Reply:
`kafka_payload` (or anything more specific) would be perfect! Great idea.